build.sh

```diff
@@ -4,7 +4,11 @@ LOG="${CWD}/log.txt"
 
 mkdir -p tmp out
 msg() {
-	echo "\033[32m[*] $@\033[0m"
+	printf "\033[32m[*] %s\033[0m\n" "$@"
```

Comment:
This has been modified because on some platforms /bin/sh is /bin/bash, and so it requires echo -e which does not exist in POSIX sh. `printf` should work fine everywhere